dds/DCPS/SafetyProfileStreams.h

```diff
@@ -20,16 +18,14 @@ OPENDDS_BEGIN_VERSIONED_NAMESPACE_DECL
 namespace OpenDDS {
 namespace DCPS {
 
-OpenDDS_Dcps_Export OPENDDS_STRING to_dds_string(::CORBA::UShort to_convert);
+OpenDDS_Dcps_Export OPENDDS_STRING to_dds_string(unsigned short to_convert);
 OpenDDS_Dcps_Export OPENDDS_STRING to_dds_string(int to_convert);
 OpenDDS_Dcps_Export OPENDDS_STRING to_dds_string(unsigned int to_convert, bool as_hex = false);
 OpenDDS_Dcps_Export OPENDDS_STRING to_dds_string(long to_convert);
 OpenDDS_Dcps_Export OPENDDS_STRING to_dds_string(long long to_convert);
 OpenDDS_Dcps_Export OPENDDS_STRING to_dds_string(unsigned long long to_convert, bool as_hex = false);
 OpenDDS_Dcps_Export OPENDDS_STRING to_dds_string(unsigned long to_convert, bool as_hex = false);
 
-OpenDDS_Dcps_Export const char* retcode_to_string(DDS::ReturnCode_t value);
```

Comment:
Just a heads up. There are some tests that include this file to get this that might break.